### PR TITLE
Async governance process

### PR DESCRIPTION
**Pull request by peterlangmar**
faster decision making, less meeting

### CODE REVIEW

File: Product Dev/Readme.md [hunk 1]
```diff
@@ -100,6 +100,10 @@ Domain:
 
 ## Domain: All functions & activities within the Circle
 
+### Policy: Async governance process
+We use the same Async governance process as TLC
```
Comment (cassus) [inline Product Dev/Readme.md]: To fully apply the async governance it also comes with an accountability for the Secretary.
Also it's nice to link to the exact policy.

You can copy-paste from https://github.com/lab-coop/lab-governance/pull/94/files

File: Product Dev/Readme.md [hunk 2]
```diff
@@ -100,6 +100,10 @@ Domain:
...
+A pull request becomes eligible for async once it is posted to #visibl on Slack
```
Comment (cassus) [inline Product Dev/Readme.md]: Did you mean #product-cirlce ?

File: Product Dev/Readme.md (hunk 1 above)
Comment (peterlangmar) [inline Product Dev/Readme.md]: :+1: added

File: Product Dev/Readme.md [hunk 3]
```diff
@@ -100,6 +102,10 @@ Domain:
 
 ## Domain: All functions & activities within the Circle
 
+### Policy: Async governance process
+We use the same Async governance process as TLC
+A pull request becomes eligible for async once it is posted to #product-circle on Slack
```
Comment (peterlangmar) [inline Product Dev/Readme.md]: @cassus yes, I've ment the product-circle, or maybe the general governance
could you manage this?

Comment (cassus) [inline Product Dev/Readme.md]: I like #product-circle 
What should I manage?

Comment (peterlangmar) [inline Product Dev/Readme.md]: I mean the auto-post to the circle's governance from github to slack

Comment (cassus) [inline Product Dev/Readme.md]: It's hard to do.
Added to my backlog as https://app.asana.com/0/35853239191384/47503119352333
No promises.

**Comment by peterlangmar** [comment]
:+1:

**Pull request merged**